src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,20 +24,23 @@
 public class RobotContainer {
 
   // The robot's subsystems
-  private final Drivetrain m_robotDrive = new Drivetrain(); // Create Drivetrain Subsystem
+  private final Drivetrain m_robotDrive;
 
   // The driver's controllers
   XboxController m_driverController = new XboxController(OIConstants.kDriverControllerPort);
   XboxController m_operatorController = new XboxController(OIConstants.kOperatorControllerPort);
 
   SendableChooser<Command> m_chooser = new SendableChooser<>();
 
-  private final DriveByController m_drive = new DriveByController(m_robotDrive, m_driverController);
+  private final DriveByController m_drive;
 
   /**
    * The container for the robot. Contains subsystems, OI devices, and commands.
+ * @param drivetrain
    */
```

Comment:
I do not understand the reason why this is in the code if it is commented out. I would recommend deleting these lines.